src/Api/Validator/Constraint/TpayChannelIdEligibility.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CommerceWeavers\SyliusTpayPlugin\Api\Validator\Constraint;
+
+use Symfony\Component\Validator\Constraint;
+
+class TpayChannelIdEligibility extends Constraint
+{
+    public const PAY_BY_LINK_CHANNEL_ID_AVAILABLE_ERROR = 'f2a42e4d-21e4-4728-a745-b49d1bf12138';
```

Comment:
I will rename the const starting with PAY_BY_LINK_CHANNEL_ID... to TPAY_CHANNEL_ID...